custom_components/ramses_cc/services.yaml

```diff
@@ -443,23 +444,23 @@ set_dhw_mode:
         boolean:
 
     duration:
-      name: Override duration
+      name: Duration
```

Comment:
Renamed so the references in the description regarding mutual exclusivity make sense.